.github/workflows/build-image.yml

```diff
@@ -0,0 +1,106 @@
+name: Build Container Image
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '39 11 * * *'
+  push:
+    branches: [ "main" ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+    paths-ignore:
+      - "README.md"
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: ${{ github.repository }}
+
+
+jobs:
+  Build_Container_NonPR:
+    if: github.event_name != 'pull_request'
```

Comment:
you can return it to how it was before, the repetition is killing me.

Reply:
bruh

Reply:
I'd argue it's better this way anyways since it's not nessecary to give the PR job the same permissions as the build one.